### PR TITLE
[Content] New adventure using new pygame keyword 'pressed'

### LINKED ISSUE #3414: [CONTENT] New adventures to explain new pygame keywords
❗ https://github.com/Felienne/hedy/issues/3337 , #3406 and #3546 have to be completed first, otherwise the new adventure won't work.

Issue #3337 adds a new keyword 'pressed' to the hedy language of level 5. In order for the students to learn about this keyword and how to use it, a new adventure should be added to level 5 with an example and explanation.

### PR DESCRIPTION
**Pull request by martijnblokker**
**Description**
This PR includes new adventures to introduce the 'pressed' keyword.
Contains story text and examples for levels 5, 8 and 9.

To check these adventures out, set language to english and look for the tab "Pressit!" in either level 5, 8 or 9.

**Fixes issue or discussion number**
Fixes #3414

### CODE REVIEW

**Comment by Felienne** [comment]
Thanks for this @martijnblokker, great that we are working on adventures for `pressed`! I would like to have adventures that tell a bit more of a story, ones that also explain *why* the keyword is useful. Something maybe with a question where you have to press A or B for a right answer? Or ASDW to control the turtle? 

I hope that makes sense!

File: content/adventures/en.yaml [hunk 1]
```diff
@@ -1571,7 +1571,28 @@ adventures:
                             turn right
                     ```
                 start_code: "# place your code here"
-
+            11:
```
Comment (Felienne) [inline content/adventures/en.yaml:1574]: Lovely adventure!! let me know when this is ready to fully review, but this looks great!!

**Comment by ToniSkulj** [comment]
> A new error appeared and that seems to be an actual issue in the code, not a typo, so "jullie zijn weer aan zet" :D

[Fix run code with pygame](https://github.com/hedyorg/hedy/pull/3698/commits/0b299b90e1e2b2bfa8675c3bae8414320a3856e0) fixes this! The pygame prefix was not used when executing the code.

**Review by Felienne — APPROVED** [review]
Well done team!!

**Pull request merged**